src/main/java/com/iexec/worker/compute/post/web2/UploaderService.java

```diff
@@ -89,38 +87,25 @@ public String uploadToIpfsWithIexecProxy(ComputedFile computedFile, String baseU
                     String.format("Can't uploadToIpfsWithIexecProxy (missing filePath to upload) [taskId:%s, fileToUploadPath:%s]", taskId, fileToUploadPath));
         }
 
-        ResultModel resultModel = ResultModel.builder()
+        final ResultModel resultModel = ResultModel.builder()
                 .chainTaskId(taskId)
                 .deterministHash(computedFile.getResultDigest())
                 .enclaveSignature(computedFile.getEnclaveSignature())
                 .zip(fileToUpload)
                 .build();
 
-        HttpHeaders headers = new HttpHeaders();
-        headers.set("Authorization", token);
+        final ResultProxyApiClient resultProxyApiClient = FeignBuilder.createBuilder(Logger.Level.NONE)
```

Comment:
I think we would like to keep basic data here to better understand what happens
```suggestion
        final ResultProxyApiClient resultProxyApiClient = FeignBuilder.createBuilder(Logger.Level.HEADERS)
```